test/system/helpers.systemd.bash

```diff
@@ -14,12 +14,44 @@ fi
 
 mkdir -p $UNIT_DIR
 
+journalctl_support_user() {
+    r_value=0
+    run systemd-analyze cat-config systemd/journald.conf | grep Storage
+    if [[ $status != 0 ]]; then
+        echo "Can not get journald.conf, don't use --user in following tests."
+    fi
+
+    if [[ "$output" == "Storage=persistent" ]]; then
+        r_value=1
+    fi
+    return r_value
+}
+
 systemctl() {
     timeout --foreground -v --kill=10 $PODMAN_TIMEOUT systemctl $_DASHUSER "$@"
 }
 
 journalctl() {
-    timeout --foreground -v --kill=10 $PODMAN_TIMEOUT journalctl $_DASHUSER "$@"
+    if is_rootless and ! journalctl_support_user; then
```

Comment:
you need to use `&&` and not `and`, it would be news to me that `and` is accepted by bash and from a quick test it doesn't work for me.